openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2085,15 +2085,9 @@ Cryptographic algorithm names used in the `credential_signing_alg_values_support
 
 The following additional Credential Issuer metadata parameters are defined for this Credential Format for use in the `credential_configurations_supported` parameter, in addition to those defined in (#credential-issuer-parameters).
 
-* `credential_definition`: REQUIRED. Object containing the detailed description of the Credential type. It consists of the following parameters defined by this specification:
+* `claims`: OPTIONAL. An array of claims description objects as defined in (#claims-description-issuer-metadata).
```

Comment:
given that `claims` is now generic across seemingly all credential formats can `claims` be added just once to the the Credential Type Metadata Parameters section?